scrapy/spiders/sitemap.py

```diff
@@ -7,6 +7,7 @@
 from scrapy.utils.sitemap import Sitemap, sitemap_urls_from_robots
 from scrapy.utils.gz import gunzip, gzip_magic_number
 
+from lxml.etree import XMLSyntaxError
```

Comment:
:lipstick: Please, move this import into a new block of third-party imports between the standard library’s and Scrapy’s. You can also move the `six` import into that block.